### PR TITLE
port rawkv tests

### PR DESCRIPTION
**Pull request by disksing**
Should be merged after #1 #2 #3

Signed-off-by: disksing <i@disksing.com>

### CODE REVIEW

File: tests/rawkv.go [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+
+type testRawKV struct{}
+
+func (t testRawKV) newCluster(ctx validator.ExecContext) *mocktikv.Cluster {
+	cluster, err := mocktikv.NewCluster(*mockTiKVAddr)
+	ctx.AssertNil(err)
+	return cluster
+}
+
+var _ = validator.Feature("rawkv.new", "create a rawkv client", nil, testRawKV{}.checkClientCreate)
```
Comment (rleungx) [inline tests/rawkv.go]: seem we need to use `RegisterFeature` right now

File: tests/rawkv.go [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+
+func (t testRawKV) checkClientCreate(ctx validator.ExecContext) validator.FeatureStatus {
+	cluster := t.newCluster(ctx)
+	defer cluster.Close()
+	client, err := stub.NewRawClientStub(*clientProxyAddr, cluster.PDAddrs())
+	if err != nil {
+		return errToFeatureStatus(err)
+	}
+	defer client.Close()
```
Comment (rleungx) [inline tests/rawkv.go:41]: Do we need to put this line before if statement?

Comment (disksing) [reply tests/rawkv.go:41]: I think not, if err is no nil, client should be nil instead, so we cannot close it.

**Review by rleungx — APPROVED** [review]

**Comment by rleungx** [comment]
BTW, does `mustSplit` in tests should be marked as TODO?

**Pull request merged**